modules/appeals_api/app/services/appeals_api/pdf_construction/supplemental_claim/v3/form_data.rb

```diff
@@ -10,12 +10,12 @@ class FormData
           BENEFIT_TYPE_CODES = {
             'compensation' => 1,
             'pensionSurvivorsBenefits' => 2,
```

Comment:
I realized I made some copy/paste typos in the data and fields in my last PR 😬 - these are fixes for that (and a few adjustments to make free text fields fill their boxes better)